docs/conf.py

```diff
@@ -455,3 +472,97 @@ def select_css(html_css_dir):
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #
 # texinfo_no_detailmenu = False
+
+
+class RedirectFrom(Directive):
```

Comment:
Is this required or was it a try to include the ROS 2 links as they were without modifying them?

Reply:
This is required to not change every file that includes the `redirect-from` directive they have.